registry/src/skills.json

```
@@ -460,5 +460,60 @@
             "Synthesizing results",
             "Preparing insights"
         ]
+    }, 
+    {
+      "pluginId": "79449cd9-0590-4ccc-9d9f-12f99438c17b",
+      "name": "@moxie-protocol/plugin-degenfans-alfafrens",
+      "displayName": "DegenFans AlfaFrens",
+      "version": "0.0.1",
+      "author": "DegenFans",
+      "description": "Enables your agent to recommend staking options for AlfaFrens channels",
+      "githubUrl": "https://github.com/moxie-protocol/moxie-agent-skills/tree/main/packages/plugin-degenfans-alfafrens",
+      "logoUrl": "https://github.com/moxie-protocol/moxie-agent-skills/tree/main/packages/plugin-degenfans-alfafrens/images/logo.png",
+      "settings": {},
+      "capabilities": ["show the best staking options for a given staking amount"],
+      "starterQuestions": [
+        {
+          "label": "What is the Degenfans Alfafrens Skills?",
+          "value": "What is the Degenfans Alfafrens Skills?"
+        },
+        {
+          "label": "What are some staking recommendations you have for Alfafrens?",
+          "value": "What are some staking recommendations you have for Alfafrens?"
+        },
+        {
+          "label": "I have 9000 AF — how should I stake it?",
+          "value": "I have 9000 AF — how should I stake it?"
+        },
+        {
+          "label": "What are your top picks for staking on Alfafrens?",
+          "value": "What are your top picks for staking on Alfafrens?"
+        }
+      ],
+      "mediaUrls": [
+        ""
+      ],
+      "actions":  [
+          "GET_ALFAFRENS_STAKING_RECOMENDATION",
+          "VIEW_ALFAFRENS_STAKING_RECOMENDATION",
+          "GIVE_ALFAFRENS_STAKING_RECOMENDATION",
+          "GET_ALFAFRENS_STAKING_CONSULTANT",
+          "STAKING_RECOMMENDATION_ON_ALFAFRENS",
+          "DEGENFANS_ALFAFRENS_INFO",
+          "WHAT_IS_DEGENFANS_ALFAFRENS",
+          "HOW_DOES_DEGENFANS_ALFAFRENS_WORK",
+          "HOW_TO_USE_DEGENFANS_ALFAFRENS"
+      ],
+      "isPremium": true,
+      "freeQueries": 0,
+      "skillCoinAddress": "0x112cf1CC540eaDF234158c0e4044C3b5F2A33e5e",
+      "minimumSkillBalance": 5,
+      "status": "ACTIVE",
+      "isDefault": false,
+      "loaders": [
+            "querying AlfaFrens",
+            "waiting for response",
+            "doing some DegenFans magic"
```

Just doing capitalization
```suggestion
            "Querying AlfaFrens",
            "Waiting for response",
            "Doing some DegenFans magic"
```